searchlib/src/vespa/searchlib/expression/filter_predicate_node.h

```diff
@@ -11,6 +11,8 @@ namespace search::expression {
 class FilterPredicateNode : public vespalib::Identifiable
 {
 public:
+    using CP = vespalib::IdentifiablePtr<FilterPredicateNode>;
```

Comment:
Although `IdentifiablePtr` is-a `CloneablePtr`, this should preferably be renamed to `IP` to avoid confusion since `IdentifiablePtr` has explicit serialization/deserialization semantics